common/build/eslint-config-fluid/src/test/mockFiles/noReleaseTagMockClass.ts

```diff
@@ -0,0 +1,39 @@
+/*!
```

Comment:
Nit: to help keep things tidy as we add new custom rules, it might be worth adding a bit of directory hierarchy for our test cases. In this case maybe just something like:
```
test-cases
- enforce-no-release-tag
- no-restricted-tag-imports 
```